pyro/distributions/transforms/neural_autoregressive.py

```diff
@@ -215,7 +113,7 @@ def _call(self, x):
         return y
 
     # This method returns log(abs(det(dy/dx)), which is equal to -log(abs(det(dx/dy))
-    def log_abs_det_jacobian(self, x, y):
+    def log_abs_det_jacobian(self, x, y, eps=1e-8):
```

Comment:
Can we make this an instance property rather than a method property? That way `.inv.log_abs_det_jacobian()` is can also customize this value.